docs/choreolib/auto-controller.md

```diff
@@ -54,13 +54,15 @@ public class AutoController implements BiConsumer<Pose2d, SwerveSample> {
             pose.getRotation()
         );
 
-        swerve.drive(out, false);
```

Comment:
Removed because a second argument for control mode (from the codebase this was ripped from) isn't particularly common- without context of that codebase, it's pretty unclear.

Reply:
Valid